### PR TITLE
#164797967 Setup webpack

### PR DESCRIPTION
**Pull request by sojida**
#### Description
This PR sets up web pack, eslint and prettier

#### Type of change
- [x] Non-Breaking Change

#### How Has This Been Tested?
N/A

#### Checklist:
- [x] setup eslint
- [x] setup prettier

#### PT-ID
[#164797967](https://www.pivotaltracker.com/story/show/164797967)

#### Screenshots
N/A

#### Questions:
N/A

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            {
+              "trailingComma": "es5",
+              "tabWidth": 2,
+              "semi": true,
+              "singleQuote": true,
+              "endOfLine": "lf"
+            }
+        ]
+    }
+}
```
Comment (walsamlee) [inline .eslintrc]: Please check the EOL

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-transform-runtime",
+  ]
+}
```
Comment (walsamlee) [inline .babelrc]: EOL here too

Comment (amaechi-chuks) [inline .babelrc]: Please fix this EOL

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Authors Haven</title>
+</head>
+<body>
+    <div id="app"></div>
+    <script src="./bundle.js"></script>
+</body>
+</html>
```
Comment (amaechi-chuks) [inline public/index.html]: You can as well fix this EOL

Comment (walsamlee) [inline public/index.html]: Another EOL needed here

File: .eslintrc (hunk 1 above)
Comment (sojida) [reply .eslintrc]: Alright. I' ll do that

File: .babelrc (hunk 2 above)
Comment (sojida) [reply .babelrc]: Alright. I' ll do that

File: public/index.html (hunk 3 above)
Comment (sojida) [reply public/index.html]: Alright. I' ll do that

File: src/index.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('app'));
+
+module.hot.accept();
```
Comment (McHardex) [inline src/index.jsx]: FYI: You can use ```"start": "webpack-dev-server --inline --hot"``` instead of module.hot.accept() here.

**Pull request merged**